src/com/from/zenhub/DivisibleBy3.java

```diff
@@ -0,0 +1,18 @@
+package com.from.zenhub;
+
+import java.util.Scanner;
+
+public class DivisibleBy3 {
```

Comment:
Change class name to DivisibleByThree from DivisibleBy3

Reply:
I had made the requested changes.